coverage/src/main/kotlin/org/rust/coverage/GrcovRunner.kt

```diff
@@ -131,5 +135,29 @@ class GrcovRunner : RsDefaultProgramRunnerBase() {
                 LOG.error(e)
             }
         }
+
+        private fun checkIsNightlyToolchain(project: Project, config: CargoCommandConfiguration.CleanConfiguration.Ok): Boolean {
+            var channel: RustChannel? = config.cmd.channel
+            if (channel == RustChannel.DEFAULT) {
+                channel = project.computeWithCancelableProgress("Fetching rustc version...") {
+                    config.toolchain.queryVersions().rustc?.channel
+                }
+            }
+            if (channel == RustChannel.NIGHTLY) return true
+
+            val option = Messages.showDialog(
+                project,
+                "Code coverage only available on the nightly channel",
```

Comment:
Maybe "Code coverage is available only with nightly toolchain" is better since now it's possible to use any nightly toolchain?